pkgs/by-name/op/optolith/package.nix

```diff
@@ -0,0 +1,111 @@
+{
+  stdenv,
+  requireFile,
+  autoPatchelfHook,
+  makeWrapper,
+  lib,
+
+  libGL,
+  udev,
+  alsa-lib,
+  at-spi2-atk,
+  cairo,
+  cups,
+  dbus,
+  expat,
+  gdk-pixbuf,
+  glib,
+  gtk3,
+  libdrm,
+  libgbm,
+  libxkbcommon,
+  nspr,
+  nss,
+  pango,
+  xorg,
+}:
+
+stdenv.mkDerivation rec {
+  name = "optolith";
+  version = "1.5.2";
+
+  src = requireFile rec {
+    name = "989002-Optolith_1.5.2.tar.gz";
+    hash = "sha256-CoGEsPxi8nZFJYdH51AgQVSrIFtlL6J6HYo8Lzp59Fg=";
+
+    message = ''
+      In order to install Optolith, you need to comply with the license agreement and download
+      the binaries, ${name}, from:
+
+      ${meta.downloadPage}
+
+      Once you have downloaded the file, please use the following command and re-run the
+      installation:
+
+      nix-prefetch-url file://\$PWD/${name}
+    '';
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    makeWrapper
+  ];
+
+  runtimeInputs = [
+    libGL.out
+    # leads to the following error if omitted:
+    # Zygote could not fork: process_type gpu-process numfds 3 child_pid -1
```

Comment:
You shouldn't need to explain in detail why inputs are needed.

Reply:
It was completely non-obvious to me that this error message has anything to do with udev, and I still have no idea why that is even the case. I only found this information by chance when googling it. If you remove libgl you'll get gl-related errors, but removing udev leads to this completely unclear message, that's why I decided to document it if just to make this issue more googlable for similar electron packaged apps.